### ISSUE TITLE
Accessibility: My Account nav missing aria-current page indicator

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43626]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The navigation menu on the My Account page does not have a current page indicator for screen reader users. Links in navigation menus should use [aria-current="page"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current) to indicate the current page.

You need to add this attribute to the currently viewed page.

Relevant code: https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/templates/myaccount/navigation.php#L25

### Expected behavior

Screen reader users will know what page is being viewed when utilizing a nav menu. 

### Actual behavior

Required aria is missing.

### Steps to reproduce

Inspect the My Account nav menu or see the code linked above.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.
